examples/dmd/local_tw_csv.cpp

```diff
@@ -272,6 +272,48 @@ int main(int argc, char *argv[])
         }
     }
 
+    // Load window RDIM/EF file if given
+    vector<string> window_str_list;
+    vector<double> window_list;
+    bool use_rdim_windows = false;
+    csv_db.getStringVector(string(list_dir) + "/" + string(window_file) + ".csv",
```

Comment:
Should there be a check for whether `window_file` is specified or exists? It has a default value, suggesting it is specified by default, but what if that file doesn't exist?